<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGTRosSprayPaintSet</class>
 <widget class="QWidget" name="QGTRosSprayPaintSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGTRosSprayPaintSet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 41, 46);</string>
  </property>
  <widget class="QGTToolButton" name="toolButton_RosSprayPaintSet_Cancel">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RosSprayPaintSet_Lidar_SearchTurning_Step">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_RosSprayPaintSet_Reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>恢复默认设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>搜寻转折点的点数步长 :</string>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_RosSprayPaintSet_Ok">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
color: rgb(85, 170, 255);
background-color: rgb(38, 76, 115);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RosSprayPaintSet_Lidar_SearchTurning_Dis_Threshold">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>搜寻转折点的距离阈值 :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RosSprayPaintSet_Lidar_SearchTurning_Len_Threshold">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>搜寻转折点的长度阈值 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_LiftPlat_InitPos">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>升降平台的初始位置 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Dis_Precision">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>工作距离的精度误差 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Chasis_Angle_Range_fromWorkwall">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-2000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>底盘与工作墙面的夹角范围 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Chasis_Dis_Threshold_fromWorkwall">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>底盘与工作墙面的距离阈值 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Chasis_MinDis_fromLeftwall">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>底盘距左边墙面的最小距离 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Chasis_MaxDis_fromLeftwall">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>底盘距左边墙面的最大距离 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Hor_WorkDis">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>单次喷涂动作水平工作距离 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Ver_WorkDis">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>单次喷涂动作垂直工作距离 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Hor_Overlap">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>水平方向的喷涂重合率 :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_RosSprayPaintSet_Ver_Overlap">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
padding:2px 4px;
background-color: rgb(255, 170, 127);
color: rgb(0, 85, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>垂直方向的喷涂重合率 :</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGTToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">Commen/qgttoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
